.github/workflows/linter.yml

```
@@ -25,9 +25,25 @@ jobs:
       - name: Install pylint
         run: pip install pylint
 
+      - name: Install mypy
+        run: pip install git+https://github.com/python/mypy.git@40bbfb5f2539f6fc3ea8c9b4de6b62d167bb003f
+
       - name: Install dependencies
         run: pip install -r requirements.txt
 
       - name: Run pylint
-        run: |
-             find -name "*.py" -not -path "./.git/*" -not -path "*/alembic/versions/*" | xargs pylint --rcfile=.pylintrc
+        run: find -name "*.py" -not -path "./.git/*" -not -path "*/alembic/versions/*" -not -path "*/tests/*" | xargs pylint --rcfile=.pylintrc
+      - name: Run mypy
+        run: find meeseeks/ -name "*.py" -not -path "./.git/*" -not -path "./tests/*" | xargs mypy
```

Лучше запускай mypy без find
```suggestion
        run: mypy --exclude=tests/ . 
```